src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -412,8 +412,8 @@ public static function debug_data() {
 				'value' => size_format( $effective ),
 			);
 			$info['wp-media']['fields']['max_file_uploads']    = array(
-				'label' => __( 'Max number of files allowed' ),
-				'value' => $max_file_uploads,
+				'label' => __( 'Max simultaneous file uploads' ),
+				'value' => number_format( $max_file_uploads ),
```

Comment:
```suggestion
				'value' => $max_file_uploads,
```
`number_format` not needed here.